css/style.css

```diff
@@ -0,0 +1,352 @@
+body {
+    margin: 0;
+}
+
+.bottom-nav{
+    text-align:left;
+    width: 95%;
+    max-width: 1200px;
+    margin: 0 auto;    
+}
+.bottom-nav-content{
+    width: 100%;
+    margin: 0 auto;
+    height: 410px;
+   
+    display: -webkit-flex;  
+    -webkit-flex-direction: row-reverse; 
+    display: flex;
+    flex-direction: row;
+}
+.bottom-nav-content section{
+    text-size-adjust: 100%;
+    float: left;
+    width: 100px;
+    height: 50px;
+    margin: 0 20px;
+    padding: 0 25px;
+ 
+}
+.go-to-homepage{
+    float: right;
+
+    
+}
+.footer-nav{
+    text-align:center;
+    width: 100%;
+}
+.footer-nav-ul{
+    display:inline-block;
+}
+.nav-li{
+    float:left;
+    margin-left:20px;
+    color: rgb(153, 153, 153);
+    font-family: nyt-franklin, helvetica, arial, sans-serif;
+    white-space: nowrap;
+    padding: 10px 0px;
+    text-decoration: none;
+    list-style: none;
+}  
+.nav-li a {
+    color: rgb(153, 153, 153);
+    font-family: nyt-franklin, helvetica, arial, sans-serif;
+    white-space: nowrap;
+    padding: 10px 0px;
+    text-decoration: none;
+    font-size: 11px;
```

Comment:
Don't forget you can use `font` shorthand where you have more than 1 font property values to set.